admin-dev/themes/new-theme/js/pages/product/product-const.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+export default {
+  PRODUCT_TYPE: {
+    STANDARD: 'standard',
+    PACK: 'pack',
+    VIRTUAL: 'virtual',
+    COMBINATIONS: 'combinations',
+  },
+};
```

Comment:
product-const.ts looks like it's not a good name, it's redundant because it's already inside the product folder, I would call it `constants` and that's it

Reply:
Ok but then it means we should also rename `product-event-map.js` `product-map.js` and so on
I'm fine with it but maybe in a future PR I just renamed the constant file for now

Reply:
Well, you're damn right about this, but then another question happens about components name 😆